clients/python/scripts/test_cloud.py

```diff
@@ -0,0 +1,54 @@
+import os
+from PIL import Image
+import moondream as md
+
+
+def test_cloud_client():
+    # Initialize client
+    api_key = os.getenv("MOONDREAM_API_KEY")
+    if not api_key:
+        raise ValueError("MOONDREAM_API_KEY environment variable not set")
+
+    client = md.VL(api_key)
```

Comment:
How does the client know this is the api_key and not the model_path? Won't it go by argument order when we don't make it a named parameter?